src/Cargo.lock

```diff
@@ -348,7 +348,7 @@ dependencies = [
 
 [[package]]
 name = "clippy_lints"
-version = "0.0.205"
```

Comment:
Could we keep `clippy_lints` at version 0.0.206 or below? Otherwise we'll be blocked by the 1.29.0 compiler bump (similar to #51122).

Reply:
Neither 0.0.205 or 0.0.207 clippy version is buildable with master so `rls` could temporary disable dependency. I think it would be simplest solution for now.
Another solution would be disabling `edition=2018` in `clippy` and fixing incompatible code, then releasing new version.

Reply:
Given that we'd need to make and keep a branch around forever, let's just wait till the 1.29 bump (it's only a few days, right?)